docs/content/querying/dimensionspecs.md

```diff
@@ -240,11 +243,12 @@ Explicit lookups allow you to specify a set of keys and values to use when perfo
 }
 ```
 
-A lookup can be of type `namespace` or `map`. A `map` lookup is passed as part of the query. A `namespace` lookup is populated on all the nodes which handle queries as per [lookups](../querying/lookups.html)
+A lookup can be of type `namespace` or `map`. A `map` lookup is passed as part of the query. 
+A `namespace` lookup is populated on all the nodes which handle queries as per [lookups](../querying/lookups.html)
 
 A property of `retainMissingValue` and `replaceMissingValueWith` can be specified at query time to hint how to handle missing values. Setting `replaceMissingValueWith` to `""` has the same effect of setting it to `null` or omitting the property. Setting `retainMissingValue` to true will use the dimension's original value if it is not found in the lookup. The default values are `replaceMissingValueWith = null` and `retainMissingValue = false` which causes missing values to be treated as missing.
```

Comment:
should be "same effect as"  (I know this was already there, but it seems worth fixing as part of this)